feature-libs/cart/saved-cart/root/events/saved-cart.events.ts

```diff
@@ -84,3 +84,11 @@ export class RestoreSavedCartFailEvent extends SavedCartEvent {
   saveCartDescription?: string;
   saveTime: string;
 }
+
+export class EditSavedCartEvent extends SaveCartEvent {}
```

Comment:
No `static readonly type`?

Reply:
nope edit and save event cart are one and the same.